src/data_inclusion/schema/modes_accueil.py

```diff
@@ -5,10 +5,10 @@ class ModeAccueil(EnhancedEnum):
     A_DISTANCE = (
         "a-distance",
         "À distance",
-        None,
+        "L'offre de service est délivrée à distance, par téléphone ou internet.",
```

Comment:
```suggestion
        "Le service est proposé à distance, par téléphone ou internet.",
```

* "Offre de service" désigne un ensemble de services et non un service particulier. Et c'est presque du jargon, qu'il faut à mon avis éviter pour s'adresser à un maximum de personnes.
* `proposé` plus simple que `délivré` + c'est moins fort : le service n'est pas exclusivement à distance, il proposé à distance et potentiellement aussi en présentiel